test/component_test.go

```
@@ -0,0 +1,119 @@
+		},
+		FromEmailAddress: &senderEmail,
+	})
+	assert.NoError(s.T(), err)
+
+	s.DriftTest(component, stack, &inputs)
+}
+
+func (s *ComponentSuite) TestEnabledFlag() {
+	const component = "ses/disabled"
+	const stack = "default-test"
+	const awsRegion = "us-east-2"
```

golangci-lint (1.62.2)

const `awsRegion` is unused

(unused)